include/net/net_if.h

```diff
@@ -664,6 +664,70 @@ struct net_if_mcast_addr *net_if_ipv6_maddr_lookup(const struct in6_addr *addr,
 						   struct net_if **iface);
 
 /**
+ * @typedef net_if_mcast_callback_t
+
+ * @brief Define callback that is called whenever IPv6 multicast address group
+ * is joined or left.
+
+ * @param "struct net_if *iface" A pointer to a struct net_if to which the
```

Comment:
Can't pass by this - I didn't see such param argument, in doublequotes and with type, before. Usually, it's just a function param name. Why do we use it here? Is it because it's a typedef, not just a function? @dbkinder - any comments?